ipaserver/install/adtrustinstance.py

```diff
@@ -482,6 +482,40 @@ def __add_cldap_module(self):
         """
         self.__add_plugin_conf('CLDAP', 'ipa_cldap', 'ipa-cldap-conf.ldif')
 
+    def __allow_trust_attr_mod(self):
+        """
+        aci to allow users to manage own trust attributes
+        """
+        # user_aci = ipalib.aci.ACI(
+        user_aci = \
```

Comment:
This function looks unnecessarily complicated. You should either be able to create the permission from ``user.managed_permissions`` dict or from a regular update file in ``./install/updates/``.

Reply:
Thanks @tiran.